readme.md

```diff
@@ -12,25 +12,43 @@
 
 ## Framing (5 minutes / 0:05)
 
-In order to do things on the client side and give our web applications behavior, we need programmatic access to the HTML and CSS using Javascript. Javascript gives us not only the ability to manipulate the **DOM**, but also to make it respond to user actions. This is where events come in: we can *listen* for certain kinds of user-driven events, such as clicking a button, entering data into a form, keypresses and many, many more.
+In order to do things on the client side and give our web applications behavior, we need programmatic access to the HTML and CSS using Javascript.
+The **DOM** not only lets us manipulate the document or webpage using JavaScript, but also gives us the ability to respond to document events using JavaScript.
+
+> What might be an event we would want to respond to with JavaScript?
+
+We can *listen* for certain kinds of user-driven events, such as clicking a button, entering data into a form, keypresses and many, many more.
 
 ### User Interaction
 
-As we write client-side Javascript -- javascript that is executed by *our browsers* -- it is very important to keep the user's actions in mind when designing our app's UI.
+As we write client-side Javascript -- javascript that is executed in *the browser* -- it is very important to keep the user's actions in mind when designing our app's UI.
 
 For example, let's say we have a single button on our landing page, we need to write some code that will execute whenever a user clicks on that button.
 
 ### What is Asynchronicity?
 
-Javascript typically will run top-to-bottom. We as developers, however, have no idea when the code related to the button click will actually be executed. It's totally dependent on the user. Therefore, we need to write code that will execute asynchronously -- or in other words, outside of the typical top-to-bottom document flow -- and not hold up the rest of our application.
+Javascript typically will run top-to-bottom. We as developers, however, have no idea when the code related to the button click will actually be executed.
+It's totally dependent on the user.
+Therefore, we need to write code that will execute asynchronously -- or in other words, outside of the typical top-to-bottom document flow -- and not hold up the rest of our application.
+
+> The term asynchronicity, like many in programming may be unduly intimidating. It simply means "not happening at the same time".
```

Comment:
Later on you have a block which talks about synchronous execution and blocking. I think it makes sense to move that chunk here above the talk about asynchronous. Reinforce what they know describing it with the new term (synchronous), then introduce asynchronous from that perspective.